std_msgs/Header header
geometry_msgs/Pose[] poses
duration[] time_from_start
